WeatherToday/Sources/Presentation/Modules/WeatherViewModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  WeatherViewModel.swift
+//  WeatherToday
+//
+//  Created by Taher on 7/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+import RxCocoa
+import RxSwift
+
+protocol WeatherViewModelInput {
+    func didFinishTyping(cityName: String)
+}
+
+protocol WeatherViewModelOutput {
+
+    var temperature: Signal<String> { get }
+    var humidity: Signal<String> { get }
+}
+
+protocol WeatherViewModelProtocol {
+
+    var input: WeatherViewModelInput { get }
+    var output: WeatherViewModelOutput { get }
+}
+
+final class WeatherViewModel: WeatherViewModelProtocol, WeatherViewModelOutput, WeatherViewModelInput {
+
+    var input: WeatherViewModelInput { self }
+    var output: WeatherViewModelOutput { self }
+
+    var temperature: Signal<String> = .empty()
+    var humidity: Signal<String> = .empty()
+
+    private let weatherUseCase: WeatherUseCaseProtocol
+    private let initialTemperatureText = "Temperature _ ℃"
```

Comment:
Optional: You can try to use `Localize.string`

Reply:
localization added in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/38dd159f8cd4d4214c6b445cfc3dd63c7149d808)